### PR TITLE
Create Buttons 

### LINKED ISSUE #1: Basic Components
- [x] Inputfield (with and without placeholder) 
- [x] Cards for the runs 
- [x] Button (Create Account, Create Run, Login)
- [x] Logo in top left as button

### PR DESCRIPTION
**Pull request by Phil9219**
Add local font to project as well

### CODE REVIEW

File: client/src/components/Button/Button.js [hunk 1]
```diff
@@ -1,55 +1,22 @@
...
+const Button = styled.button`
+  height: 40px;
+  width: 330px;
+  border-radius: 15px;
+  background: var(--primary-color);
+  border: 2px solid var(--secondary-color);
+  cursor: pointer;
+  color: var(--secondary-color);
+  font-size: 29px;
+  font-family: "Heiti SC", sans-serif;
```
Comment (kschmitz123) [inline client/src/components/Button/Button.js:14]: you don't need this here once you set the font-family in the globalStyle

File: client/src/components/Button/Button.js [hunk 2]
```diff
@@ -1,55 +1,22 @@
...
+`;
+export const Buttons = ({ title }) => {
```
Comment (kschmitz123) [inline client/src/components/Button/Button.js:16]: the naming is a bit confusing because you''re only exporting one button

**Comment by kschmitz123** [comment]
lgtm 👍

File: client/src/components/Button/Button.js (hunk 1 above)
Comment (Phil9219) [reply client/src/components/Button/Button.js:14]: only font-family is set in globalStyle.
Cursor, color and font-size are only set in Button.js. Is there something I don´t see?

Comment (kschmitz123) [reply client/src/components/Button/Button.js:14]: i just meant font-family :)

**Pull request merged**